bridetalk_project/src/components/pages/login/loginpage.css

```diff
@@ -0,0 +1,27 @@
+.login-container {
+  width: 30%;
+  background-color: hsl(346, 58%, 98%);
+  color: hsl(0, 11%, 0%);
+  height: 80%;
+  padding: 20px 10px 40px 10px;
+  border-radius: 35px;
+  display: block;
+  text-align: center;
+  margin: 60px auto;
+}
+
+.login-fields-container {
+  text-align: start;
+  display: block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.forgot-password {
+  margin-left: 20%;
+  margin-bottom: 40px;
+}
+
+a {
```

Comment:
I will let you read an article about it instead of me explaining it: https://frontstuff.io/you-need-to-stop-targeting-tags-in-css 

I know it is outside of the scope of the project, and probably noone will have a problem with u doing it like that. But it is a rule of thumb to avoid targeting HTML Tags in CSS. It can cause tones of problem as an application scale, and as you saw the other day with the same class naming that used to override your styles (.text), this can be a lot worse. Give a meaningful classname to the elements, target the classnames instead.